com.unity.multiplayer.mlapi/Runtime/Serialization/NetworkWriter.cs

```diff
@@ -233,7 +233,7 @@ public void WriteObjectPacked(object value)
             }
 
 
-            throw new ArgumentException($"{nameof(NetworkWriter)} cannot write type {value.GetType()}");
+            throw new ArgumentException($"{nameof(NetworkWriter)} cannot write type {value.GetType().Name} - it does not implement INetworkSerializable");
```

Comment:
Ok, I know this doesn't belong in this PR but this helped me and I don't want to forget to include it (it's printing the wrong thing)

Reply:
I think that's OK, I'd suggest a minor change though :)

```suggestion
            throw new ArgumentException($"{nameof(NetworkWriter)} cannot write type {value.GetType().Name} - it does not implement {nameof(INetworkSerializable)}");
```